src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationLambdaInEnum.java

```diff
@@ -0,0 +1,87 @@
+/* Config:                                                                          //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration:         //indent:1 exp:1
+ * tabWidth = 8                                                                     //indent:1 exp:1
+ * basicOffset = 4                                                                  //indent:1 exp:1
+ * braceAdjustment = 0                                                              //indent:1 exp:1
+ * caseIndent = 4                                                                   //indent:1 exp:1
+ * throwsIndent = 4                                                                 //indent:1 exp:1
+ * forceStrictCondition = false                                                     //indent:1 exp:1
+ * lineWrappingIndentation = 4                                                      //indent:1 exp:1
+ */                                                                                 //indent:1 exp:1
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+import java.util.function.Predicate; //indent:0 exp:0
+
+import java.util.List; //indent:0 exp:0
+
+public enum InputIndentationLambdaInEnum { //indent:0 exp:0
+	/** //indent:8 exp:8
+	 * Getter. //indent:9 exp:9
+	 * T get() //indent:9 exp:9
+	 */ //indent:9 exp:9
+	GET(-1, false, 1, m -> m.isEmpty() //indent:8 exp:8
```

Comment:
if basic offset is 4 , why expected indentation is 8 ?

Reply:
```
src % cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name = "Checker">
    <property name="charset" value="UTF-8"/>

    <property name="severity" value="warning"/>

    <property name="fileExtensions" value="java, properties, xml"/>

    <module name="TreeWalker">
        <module name="Indentation">
        </module>
    </module>

</module>    
```
```
src % cat Test.java 
import java.util.List;
import java.util.function.Predicate;

enum Test {
LOW,
        MEDIUM,
                        HIGH
}


enum Test2 {
GET0(false, m -> m.isEmpty()),
    GET1( false, m -> m.isEmpty()),
        GET2( false, m -> m.isEmpty()),
            GET3( false, m -> m.isEmpty()),
                GET4( false, m -> m.isEmpty());


    private final Predicate<List<?>> detector;
    private final boolean multi;

    Test2(boolean multi, Predicate<List<?>> detector) {
        this.multi = multi;
        this.detector = detector;
    }
}
```                                                                                                                                                                
```
src % java -jar checkstyle-10.0-all.jar -c config.xml Test.java
Starting audit...
Audit done.
```

Might be existing bug with enum

Reply:
@romani ping

Reply:
But in issue that you fix there is no violation on enum value, indentation is not 8.

Reply:
@romani

Reply:
Please help me to find in your input file a case that proves that on code that was reported in issue there will be no violations.

https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/4cb4a45_2022044222/reports/diff/Vavr/index.html#A2 This is bad message, requested indentation is wrong.

Reply:
https://github.com/checkstyle/checkstyle/pull/11306/files/a8a431c40dc9eb28a8267fec8d46e24ea0cd9bce#diff-799fdd58c9b9c5e9680e0ec155fbe632ac62b30e352f5c0a2e367d60a694519aR16-R25

In the issue, it was causing a loop if indent was 8, expected was 12, indent 12 expected 16...

Reply:
> https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/4cb4a45_2022044222/reports/diff/Vavr/index.html#A2 This is bad message, requested indentation is wrong.

https://github.com/checkstyle/checkstyle/issues/5448#issuecomment-925595615

@romani 
Here is the similar issue, in this indent is correct but still it suggest indent to be similar with previous line, it is not able to check correctly when there is linewrapping

Reply:
@romani  ping